### DISCUSSION TITLE
可有考虑支持英文？

### DISCUSSION THREAD

**Discussion by clasy0947** [AlistGo/alist#1229] [Ideas]
可有考虑支持英文？

**Comment by anwen-anyi** [comment]
什么英文?语言吗？ 右下角悬浮栏不是有英文吗QAQ

**Comment by vbskycn** [comment]
***@***.***
 
发件人： clasy0947
发送时间： 2022-06-14 17:20
收件人： alist-org/alist
抄送： Subscribed
主题： [alist-org/alist] 可有考虑支持英文？ (Discussion #1229)
可有考虑支持英文？
